Utils/PluginActivator/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+
+# If not stated otherwise in this file or this component's license file the
+# following copyright and licenses apply:
+#
+# Copyright 2020 Metrological
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cmake_minimum_required(VERSION 3.10.3)
+
+project( PluginActivator )
+
+set( CMAKE_CXX_STANDARD 14 )
```

Comment:
Wished we could :-) We still have customers/platforms where the best we can do is c++11, so we (for maximum portability) only allow std C++ 11.
If we need to eleborate, lets put it on the agende for next week...

Reply:
Switched to C++ 11 - not utilising any C++14 specific features anyway